PR_CHECKLIST.md

```diff
@@ -0,0 +1,56 @@
+# Pull Requests Checklist
+
+This is the checklist that mantainers go through for every single PR that they review.
+If you're wondering why it takes so long for them to accept pull requests, this is why.
+
+## General
+
+- Is this change useful to me, or something that I think will benefit others greatly?
+- Check for overlap with other PRs.
+- Think carefully about the long-term implications of the change. How will it affect
+  existing projects that are dependent on this? How will it affect my projects? If
+  this is complicated, do I really want to maintain it forever? Is there any way it
+  could be implemented as a separate package, for better modularity and flexibility?
+
+## Check the Code
+
+- If it does too much, ask for it to be broken up into smaller PRs.
+- Does it pass the lint stage on our CI?
+- Is it consistent?
+- Review the changes carefully, line by line. Make sure you understand every single
+  part of every line. Learn whatever you do not know yet.
+- Take the time to get things right. PRs almost always require additional improvements
+  to meet the bar for quality. Be very strict about quality. This usually takes several
+  commits on top of the original PR.
+
+## Check the Tests
+
+- Does it have tests? If not:
+  - Comment on the PR "Can you please add tests for this code to test_blah.py", or...
+  - Write the tests yourself.
+
+- Do the tests pass on our CI? (aka do they work on all our supported
+  platforms?)
+
+## Check the Docs
+
+- Does it have docs? If not:
+  - Comment on the PR "Can you please add docs for this feature to docs/usage.rst",
+    or...
+  - Write the docs yourself.
+
+- If any new functions/classes are added, do they contain docstrings?
+
+- If any new features are added, are they in README.rst?
+
+## Credit the Authors
+
+- Add name and URL to AUTHORS.rst.
+- Thank them for their hard work.
+
+## Close Issues
+
+- Merge the PR branch. This will close the PR's issue.
+- Close any duplicate or related issues that can now be closed. Write thoughtful
+  comments explaining how the issues were resolved.
+
```

Comment:
Hey @glumia, I think we should at least add credit to @audreyfeldroy because it is not a part of cookiecutter itself. I think is better to include this new file in another pr and wait for the approval.

What do you think?

Reply:
Yes, makes sense! I'll add credit in the `PR_CHECKLIST.md` file itself and move its commit in a separate PR so we can merge 4c5d53b in the meanwhile.